bigquery/bigquery_storage_quickstart/main_test.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+package main
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestApp(t *testing.T) {
+	m := testutil.BuildMain(t)
+	defer m.Cleanup()
+
+	if !m.Built() {
+		t.Errorf("failed to build app")
+	}
+
+	env := make(map[string]string)
+	if p, ok := os.LookupEnv("GOLANG_SAMPLES_PROJECT_ID"); ok {
```

Comment:
Use `testutil.SystemTest`, which will skip the test if `GOLANG_SAMPLES_PROJECT_ID` is not set.

Reply:
done